tests/containers/host_configuration.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Setup system which will host containers
+# - setup networking via dhclient
+# - make sure that ca certifications were installed
+# - import SUSE CA certificates
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use utils;
+use scheduler 'get_test_suite_data';
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+    my $test_data = get_test_suite_data();
+    assert_script_run "dhclient -v" if $test_data->{host_os} eq 'centos';
+    ensure_ca_certificates_suse_installed() unless $test_data->{host_os};
+    assert_script_run "curl http://ca.suse.de/certificates/ca/SUSE_Trust_Root.crt -o /etc/ssl/certs/SUSE_Trust_Root.crt";
```

Comment:
I don't understand this. I'd do something like:
```perl
if (is_sle() || is_opensuse()) {
    ensure_ca_certificates_suse_installed() unless $test_data->{host_os};
} else {
    assert_script_run "curl http://ca.suse.de/certificates/ca/SUSE_Trust_Root.crt -o /etc/ssl/certs/SUSE_Trust_Root.crt";
}
```

Reply:
Think of it as "if `$test_data->{host_os}` is other than is_sle/is_openesuse" The extra if/else do not make any difference here. `is_sle` and `is_opensuse` will be always true (unless jeos)

Reply:
what about the `curl` command? I guess we want to execute that only in non-suse OSs, right?

Reply:
That sounds right. I will adjust.